core/eval.go

```diff
@@ -187,6 +187,19 @@ func evalSET(args []string) []byte {
 			if obj != nil {
 				return RESP_NIL
 			}
+		case "GET":
+			obj := Get(key)
```

Comment:
You can just call evalGET() method before the value has been updated and return the o/p of evalGET()  as is. wdyt?

Reply:
Okay, so, like, I've to use `evalGET()` function instead of `Get()`, am I right?

Reply:
@soumya-codes , I've a doubt, that since by using the `evalGET` function, we need to pass an array <br>
![image](https://github.com/user-attachments/assets/acfc3b23-2548-4d0f-998b-01af83da442f)

So, first, we've to push the key to an array and then pass it to the `evalGET` func, which will increase the fetching time, so can we just let it be `Get()` only? because in other cases also, the `Get()` function is only used!
What say?